book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
Great job so far. Please add a method that adds `Rentals` to `Book` since `Book` has many `Rentals`. It is also one of the project requirements. 
![image](https://github.com/lerfast/opp-school-library/assets/101523582/42a8b4e0-fa2e-4d46-826e-08fbc9c8bd43)

You can visit this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more.
Kindly, add it.

Reply:
![image](https://github.com/lerfast/opp-school-library/assets/111823606/91748ea8-20ee-4f8e-acf0-bfab766cad25)
Got it!!!